client/extensions/woocommerce/app/promotions/fields/applies-to-single-product-field.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import FormField from './form-field';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import { getProductCategories } from 'woocommerce/state/sites/product-categories/selectors';
+import { getPromotionableProducts } from 'woocommerce/state/selectors/promotions';
+import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
+import Search from 'components/search';
+
+function productContainsString( product, textString ) {
+	const matchString = textString.trim().toLowerCase();
+
+	if ( -1 < product.name.toLocaleLowerCase().indexOf( matchString ) ) {
```

Comment:
Just curious why toLocalLowerCase wasn't used to generate matchString as well?

Reply:
Could just return the conditional

Reply:
I noticed that too, but I think I changed it in the next PR to use `toLocaleLowerCase()` on both.

Reply:
> Could just return the conditional
Yeah, I suppose... I guess I get nervous about returning numerics as booleans.